chall03/krusthol.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func log_prefixed(msg string) {
+	fmt.Printf("%d", time.Since(StartTime).Milliseconds())
+	log.Println("ms - ", msg)
+}
+
+func get_request(address string) (id_rgb_str string) {
```

Comment:
I would not recommend using a function to do the http request, because if you need to use, one day, a response Header or StatusCode in your main scope, you will need to return some parameters, and start to make the code less cleaner.